project-templates/import-records/format-people-config.js

```diff
@@ -0,0 +1,107 @@
+var normalize = require('medic-bulk-utils').normalize,
```

Comment:
The import process is basically format and then input.  Sometimes we don't get the formats we want from the data provider (client db export) so we have a format phase before import.  Each type needs a format step atm, this will get streamlined a little more though eventually.